data/repository/src/main/java/com/chocolate/repository/model/dto/direct_message/Chat.kt

```diff
@@ -0,0 +1,10 @@
+package com.chocolate.repository.model.dto.direct_message
+
+import java.util.Date
+
+data class Chat(
```

Comment:
rename the model to `ChatDto`
rename the package from `direct_message` to `directmessage` and read this please [Package Names](https://developer.android.com/kotlin/style-guide#package_names)